.github/workflows/test-unit.yml

```diff
@@ -5,63 +5,35 @@ on:
 
 env:
   GO_VERSION: 1.22.2
-  TEST_PARALLELISM: 6
+  TEST_PARALLELISM: 12
   TEST_PARALLELISM_PKG: 6
```

Comment:
Should we increase paralellism of unit test packages? Looks like the tests are not running faster.

Reply:
Unit test are using testing project in a small number of tests, CLI E2E tests use a testing project in each test.

Reply:
ok. So I just increased the `TEST_PARALLELISM`: 6 -> 12. `TEST_PARALLELISM_PKG` will remain 6.